doc/sphinx/proof-engine/tactics.rst

```diff
@@ -1718,40 +1701,35 @@ Controlling the proof flow
              name the variable in the current goal and in the context of the
              existential variable. This can lead to surprising behaviors.
 
+   .. deprecated:: 8.16
+
+      The no argument form is equivalent to :tacn:`idtac`.  This form (only)
+      is deprecated in 8.16.
+
    The second form refines an existential variable selected by its position.  The
    :n:`@natural` argument is the position of the existential variable
-   *from right to left* in the goal.  (Use one of
-   the variants below to select an existential variable in a
+   *from right to left* in the goal.  (Use the :n:`@hloc` clause
+   to select an existential variable in a
    hypothesis.)  Counting starts at 1 and multiple occurrences of the
-   same existential variable are counted multiple times.  Because this
-   variant is not robust to slight changes in the goal, its use is
-   strongly discouraged.
+   same existential variable are counted multiple times.  Using this form
+   is discouraged because slight changes to the goal may change the needed index,
+   causing a maintenance issue.
 
    :n:`in @ident`
      Selects the hypothesis :n:`@ident`.
 
    :n:`in %|- *`
-     Selects all hypotheses and the goal.
-
-   .. todo PR above is correct (in |- *)?
+     Selects the goal.  This is the default behavior.
 
    :n:`in ( type of @ident )`
-    Selects existential variables with the type :n:`@ident` in a
-    :term:`local definition <context-local definition>`
+     Selects existential variables in the type given in the
```

Comment:
The choice of "a" rather than "the" is just to be consistent with what you've put below. The most important part is writing "type of".

```suggestion
     Selects existential variables in the type of a
```

Reply:
"type of", got it.  I think "the" is appropriate because ```:term:`local definition <context-local definition>` :n:`@ident` ``` identifies a specific local definition.